docs/Project-Configuration/Project-Commands.md

```diff
@@ -1453,9 +1489,13 @@ Parameters:
     you can use this parameter.
 -   `visibility`: "public" (default) which provides a link to the
     s3 path in the UI for all Evergreen users. "private" which is a legacy option that now does the
-    same as "public". "none" which hides the file from the UI for everybody.
-    "signed" which creates a pre signed url with the provided static credentials, allowing users to see the file (even if it's private on S3). Visibility: signed should not be combined with permissions: public-read or permissions: public-read-write, or aws_session_token.
-    Note: This parameter does not affect the underlying permissions of the file
+    same as "public". "none" which hides the file from the UI for everybody but does not
+    affect the underlying s3 permissions (see `permissions` parameter). "signed" which creates
+    a pre signed url with the provided role_arn or credentials, allowing users to see the file
+    (even if it's private on S3). Visibility: signed should not be combined with
+    permissions: public-read or permissions: public-read-write. It can be combined with aws_session_token
+    but it will only work if the generated credentials are from a previous `ec2.assume_role` command in this task.
```

Comment:
Just to check, this means if a user manually generates temporary credentials (e.g. using the AWS CLI or something), it can't be used with `visibility: signed`, correct? Is that because temporary credentials created without `ec2.assume_role` are roles that we may not be able to assume when it comes time to sign the file?